### DISCUSSION TITLE
User Research plan

### DISCUSSION THREAD

**Discussion by lpatmo** [codebuddies/frontend#167] [General]
@wuworkshop pointed to https://creativegood.com/blog/20/customer-research-quarantine.html

**Comment by BethanyG** [comment]
I would add (for codebuddies) to what is detailed there:

- ask the user what sites/products/meetups **they use** that do same things they would do on the codebuddies website
- if the user is not aware of the website, ask them what sites they think of or go to for similar activities, and what they like about them.  Have them demonstrate, if they can.
- ask them if there are things they would like to see _presented_ differently or _implemented_ differently in the sites they use for similar activities, and why they feel that way.  If they are aware of the codebuddies website, ask them the same thing about codebuddes.
- ask them which activities they do on chat that they *do not* do via discussion forum or email.  
- ask them which activities they do via "hangout" or "pairing" that they wouldn't do via chat or discussion group.
- ask them what sites/books/chats/meetups/strategies they would recommend to a developer who wants to do different activities (like paring, discussions, algo review, interview prep, self-study, etc.)

**Reply by lpatmo** [reply]
Love these questions!
